### ISSUE TITLE
Exported SVG shapes with an inner stroke have a center stroke instead

### ISSUE THREAD

**Issue by break-core** [photopea/photopea#5580]
# TL;DR 
Photopea won't export SVG shapes with an inside stroke properly, instead will revert back to a center stroke.

# Explanation:
In Photopea, with the shape tool or pen tool, you are able to freely set a stroke on a shape. Additionally, you can change the positioning of the stroke by clicking on the line button next to the text box to change stroke size. This is good, but the problem arises when exporting an SVG.

If you export a shape with a stroke that is inside as an SVG, then Photopea will not export properly. Instead of having a stroke positioned inside of the shape's lines, it will instead position it to the center of the shape's lines. This results in an inaccurate design.

# Reproduction Steps:
Step 1: Open a normal blank canvas on Photopea
Step 2: Use the pen tool and create a shape
Step 3: Create a stroke of any size on the shape AND set the position of it to INSIDE
Step 4: Select the shape you want to export, go to the move tool and press the `Download` button next to the Distances toggle.
Step 5: Hit the SVG button (you can change the export scale to whatever you want)
Step 6: Preview the SVG in any editor and do a comparison. You should see that the SVG does not have an inner stroke, rather an outer stroke.

# Bug In Action:
`Pre-Export`
![image](https://user-images.githubusercontent.com/71668864/235323729-39066407-8342-4c03-87fa-37f364dd4881.png)

`Post-Export`
![image](https://user-images.githubusercontent.com/71668864/235323784-190f54ef-4240-4856-89df-81c7a153097f.png)

**Comment by photopea** [comment]
Do you think there can exist an SVG files with an inner stroke? Could you show us an SVG file with an inner stroke?

**Comment by break-core** [comment]
> Do you think there can exist an SVG files with an inner stroke? Could you show us an SVG file with an inner stroke?
I definitely think there can be. Many people have wished for stroke alignment to be a proposed feature in SVG, but so far none have necessarily turned up. Though, there is an fxtf draft which has stroke align in mind: https://drafts.fxtf.org/fill-stroke-3/#propdef-stroke-align

There are some hacks that some have done to achieve an inner stroke on SVGs. One of the main ones would be trying to use a [clipped shape mask](https://alexwlchan.net/2021/inner-outer-strokes-svg/), or alternatively try and draw the stroke as it's own shape.

Though if there would be a more authentic implementation, then we'd have to wait for stroke-align to be added in.
